src/igvfd/tests/data/inserts/multiplexed_sample.json

```diff
@@ -8,6 +8,10 @@
             "igvf:multiplex_alias"
         ],
         "barcode_map": "igvf:tabular_file_04_barcode_to_sample_mapping",
+        "multiplexing_method": [
+            "genetic",
+            "barcode based"
+        ],
```

Comment:
should we use plural form in property name (multiplexing_methods), since it's an array and we allow it to have multiple items?